### ISSUE TITLE
Focus on tabliss search bar when opening a new tab

### ISSUE THREAD

**Issue by wa4-fearless-otter** [joelshepherd/tabliss#137]
When opening a new tab in Firefox the address bar is focused and not the tabliss search bar. I don't think this should be the case.

**Comment by kadmil** [comment]
And as another tabliss user, I'd say that I would hate having something other than browser address bar focused on the new page. All in all, it's where my default search lives, with all the hints and search settings, which I don't ever want to have to set up again.

**Comment by wa4-fearless-otter** [comment]
I think this should be setable. So you can activate it if you need it

**Comment by hematogender** [comment]
This is a default Firefox behavior. Even for their default start page with a search bar lol... I researched this problem for a few hours and couldn't find the working solution, except the following.

Maybe we could insert ``document.getElementsByClassName("Search")[0].remove();`` in index.html? It works from the console, but I'm not experienced enough, nor have time, sadly, to fire up my own fork of this repo. Would be cool if someone with proper skills tested it.

**Comment by gennaro-tedesco** [comment]
I support the idea of having auto-focus on the Tabliss search bar (or at least setable). After all, the entire point of having a search bar in a new tab is because you want to avoid clicking somewhere after opening the new tab (I could just click on google otherwise).

**Comment by BrunoGomesCoelho** [comment]
TLDR: Also quite like this, nothing I tried worked :)

I want to use Firefox in full-screen mode and use tabliss search instead of Firefox when opening a new tab; If the focus isn't on the address bar, it works as intended.

Since Custom JS doesn't seem to be available in the local installation (#90 ), what I tried was using the web version of Tabliss with the code snippet above; That does work as intended on full screen, but I haven't been able to set that as a default new tab; Extensions such as [Custom New Tab Page](https://addons.mozilla.org/en-US/firefox/addon/custom-new-tab-page/) seems to keep the focus on the address bar anyway :/

**Comment by Typhonling** [comment]
This is not a good idea, not everyone likes the search box and it does not work with custom search engines.
If the focus can be customized, that would certainly be best. The correct default should be that the focus is on the search box when it is enabled, otherwise it should remain on the address bar.
